docs/sources/alerting/unified-alerting/alerting-rules/create-cortex-loki-managed-rule.md

```diff
@@ -1,40 +1,40 @@
 +++
-title = "Create Cortex or Loki managed alert rule"
-description = "Create Cortex or Loki managed alerting rule"
+title = "Create Mimir or Loki managed alert rule"
+description = "Create Mimir or Loki managed alerting rule"
 keywords = ["grafana", "alerting", "guide", "rules", "create"]
 weight = 400
 +++
 
-# Create a Cortex or Loki managed alerting rule
+# Create a Mimir or Loki managed alerting rule
 
-Grafana allows you to create alerting rules for an external Cortex or Loki instance.
+Grafana allows you to create alerting rules for an external Mimir or Loki instance.
 
 ## Before you begin
 
-- Verify that you have write permission to the Prometheus data source. Otherwise, you will not be able to create or update Cortex managed alerting rules.
+- Verify that you have write permission to the Prometheus data source. Otherwise, you will not be able to create or update Mimir managed alerting rules.
```

Comment:
```suggestion
- Verify that you have write permission to the Prometheus data source. Otherwise, you will not be able to create or update Grafana Mimir managed alerting rules.
```